client/app/profile/profile.controller.js

```diff
@@ -22,6 +22,8 @@
         vm.errorSetRole = false;
         vm.errorSetLevel = false;
         vm.errorSetContactDetails = false;
+        vm.verificationEmailSent = false;
+        vm.errorVerificationEmailSent = false;
```

Comment:
Is this var needed?  Or is it required to keep state locally??  If it's required maybe add a comment to explain it's purpose.